tests/misc/non_compliant.py

```diff
@@ -118,10 +118,10 @@
     print('NotImplementedError')
 
 # struct pack with too many args, not checked by uPy
-print(ustruct.pack('bb', 1, 2, 3))
+print(struct.pack('bb', 1, 2, 3))
```

Comment:
We should probably not pass this test because its checking that MicroPython doesn't match CPython.

Reply:
ok, another difference